Collect code coverage

ligurio opened the pull request:
Enable integration of luacov with luatest and show
code coverage statistics once regression tests passed.

Closes #43

Totktonada commented on .github/workflows/test.yml:
```diff
@@ -39,9 +42,13 @@ jobs:
       - run: tarantoolctl rocks remove ddl --force
 
-      - run: luacheck .
+      - name: Run luatest
+        run: luacheck .
```
The name is 'Run luatest', but we run luacheck here.

ligurio replied:
Fixed

ligurio commented:
@rosik, I have fixed output and added a couple of related commits to PR (fixed typo and fixed cmake's warnings).
Could you review again and do **Rebase and Merge** to avoid squashing commits?
